SofaKernel/modules/Sofa.GL/CMakeLists.txt

```diff
@@ -83,14 +83,14 @@ endif()
 # own include dir, the macro does not handle "independent" package for SofaFramework-style include 
 # i.e NOT the name of package first (sofa/gl is wanted, instead of Sofa.GL/)
 target_include_directories(${PROJECT_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/include/${PROJECT_NAME}/${PROJECT_NAME}>")
+target_include_directories(${PROJECT_NAME} PUBLIC "$<INSTALL_INTERFACE:include/${PROJECT_NAME}/${PROJECT_NAME}>")
```

Comment:
Why that? Sofa.GL headers are installed in `Sofa.GL/include/Sofa.GL/sofa/gl/...`?
If yes, it shouldn't it be `Sofa.GL/include/sofa/gl/...` instead?

Reply:
Oh, it's not relocatable, I see now :+1:

Reply:
See my commit.